### DISCUSSION TITLE
Is it possible to predict the proportion of drug uptake in different tissues in PK-SIM?

### DISCUSSION THREAD

**Discussion by RoachWang** [Open-Systems-Pharmacology/Forum#1571] [Questions & Problems] [answered]
Can I predict, by PK-SIM, the percentage of drug absorption in different tissues, especially in different intestinal segments?
If so, can anyone provide a reference or how-to guide?
Much appreciated.

**Comment by Ricardo-DO** [comment, marked as answer]
Hi RoachWang

I think you can do it.
When selecting the outputs of your simulation, go to Compartment->Lumen
and select Fraction absorbed to the mucosa for each segment

![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/59565610/e9c8b038-fdbd-4351-bcbb-4adac87282e2)

![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/59565610/7ce27326-2656-40f2-8f43-441a7c760b39)

Then just export the data to Excel and look for the plateau value, and that should be the fraction absorbed in that region

![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/59565610/c5f5c357-ff32-4fcc-ad52-d1364aaf01bb)

Hope it helps,
Ricardo

**Reply by RoachWang** [reply]
Thank you very much for your answer!
